src/components/panel/panel.js

```diff
@@ -869,15 +869,12 @@ MdPanelRef.prototype.close = function() {
         return self.detach();
       }, this._$q.reject)
       .then(function() {
```

Comment:
I think this then chain is unnecessary. It tests fail when you remove it, go ahead and leave it. But I think you should be okay.